CHANGELOG.md

```diff
@@ -1,3 +1,41 @@
+## v1.7.0 (06/15/2020)
+
+**Common**
+
+* Corrected various formatting to be compliant with the coding style.
+* Fixed the issue with `olp::client::TaskContext`. Now, it does not trigger cancellation after a successful operation.
+* Added the `default_cache_expiration` added to `olp::client::OlpClientSettings`. Now, you can can control the time how long the downloaded data is considered valid. Once the data has expired it will not be returned to the user upon requests and gradually removed from the cache when new data is added.
```

Comment:
```suggestion
* Added the `default_cache_expiration` to `olp::client::OlpClientSettings`. Now, you can can control the time how long the downloaded data is considered valid. Once the data has expired it will not be returned to the user upon requests and gradually removed from the cache when new data is added.
```